### PR TITLE
PR: feat/db-migration into Dev

### PR DESCRIPTION
**Pull request by lpendem**

### CODE REVIEW

File: controllers/permits/applications/special-uses/commercial/outfitters/index.js [hunk 1]
```diff
@@ -110,9 +120,48 @@ const post = function(req, res){
 		const response = include('test/data/outfitters.post.json');
 
 		response.apiRequest = postData;
-    
-		res.json(response);
-    
+
+		// api database updates
+		const controlNumber = Math.floor((Math.random() * 10000000000) + 1);
```
Comment (cameronwolf) [inline controllers/permits/applications/special-uses/commercial/outfitters/index.js]: Might be good to check for a collision on the controlNumber, and handling it if there is one.

File: controllers/permits/applications/special-uses/commercial/outfitters/index.js [hunk 2]
```diff
@@ -110,9 +120,48 @@ const post = function(req, res){
...
+			website = postData.applicantInfo.website;
+		}
+
+		dbUtil.saveApplication(controlNumber, postData.tempOutfitterFields.formName, website, function(err, appl) {
+
+			if (err) {
+				error.sendError(req, res, 400, 'error saving application in database');
+			}
+			else {
+				dbUtil.saveFile(appl.id, 'inc', postData.tempOutfitterFields.insuranceCertificate, function(err, file) {
```
Comment (cameronwolf) [inline controllers/permits/applications/special-uses/commercial/outfitters/index.js]: If the plan isn't to use file, might consider removing it.

File: controllers/permits/applications/special-uses/commercial/outfitters/index.js (hunk 1 above)
Comment (lpendem) [inline controllers/permits/applications/special-uses/commercial/outfitters/index.js]: @cameronwolf control number will be generated by the basicAPI in SUDS, this is just workaround for now.

Comment (cameronwolf) [inline controllers/permits/applications/special-uses/commercial/outfitters/index.js]: I figured but just wanted to make sure we addressed it, if it needed to be.

**Pull request merged**